opus/application/test_api/test_ui_api.py

```diff
@@ -2371,8 +2451,8 @@ def test__api_normalizeurl_real_5(self):
 
     def test__api_normalizeurl_real_6(self):
         "[test_ui_api.py] /__normalizeurl: real 6"
-        url = '/opus/__normalizeurl.json?SURFACEGEOplutocenterresolution2=5&SURFACEGEOplutophase1=160&surfacegeometrytargetname=Pluto&qtype-SURFACEGEOplutophase=any&cols=opusid,instrument,time1,SURFACEGEOplutoplanetographiclatitude1,SURFACEGEOplutoplanetographiclatitude2,SURFACEGEOplutoIAUwestlongitude1,SURFACEGEOplutoIAUwestlongitude2,SURFACEGEOplutocenterdistance,SURFACEGEOplutocenterresolution,SURFACEGEOplutophase1,SURFACEGEOplutophase2&widgets=SURFACEGEOplutocenterresolution,SURFACEGEOplutophase,surfacegeometrytargetname&order=time1,opusid&view=search&browse=gallery&cart_browse=gallery&startobs=1&cart_startobs=1&detail='
-        expected = {"new_url": "surfacegeometrytargetname=Pluto&SURFACEGEOplutocenterresolution2=5&unit-SURFACEGEOplutocenterresolution=km/pixel&SURFACEGEOplutophase1=160&qtype-SURFACEGEOplutophase=any&unit-SURFACEGEOplutophase=degrees&cols=opusid,instrument,time1,SURFACEGEOplutoplanetographiclatitude1,SURFACEGEOplutoplanetographiclatitude2,SURFACEGEOplutoIAUwestlongitude1,SURFACEGEOplutoIAUwestlongitude2,SURFACEGEOplutocenterdistance,SURFACEGEOplutocenterresolution,SURFACEGEOplutophase1,SURFACEGEOplutophase2&widgets=SURFACEGEOplutocenterresolution,SURFACEGEOplutophase,surfacegeometrytargetname&order=time1,opusid&view=search&browse=gallery&cart_browse=gallery&startobs=1&cart_startobs=1&detail=", "new_slugs": [{"surfacegeometrytargetname": "Pluto"}, {"SURFACEGEOplutocenterresolution2": "5"}, {"unit-SURFACEGEOplutocenterresolution": "km/pixel"}, {"SURFACEGEOplutophase1": "160"}, {"qtype-SURFACEGEOplutophase": "any"}, {"unit-SURFACEGEOplutophase": "degrees"}, {"cols": "opusid,instrument,time1,SURFACEGEOplutoplanetographiclatitude1,SURFACEGEOplutoplanetographiclatitude2,SURFACEGEOplutoIAUwestlongitude1,SURFACEGEOplutoIAUwestlongitude2,SURFACEGEOplutocenterdistance,SURFACEGEOplutocenterresolution,SURFACEGEOplutophase1,SURFACEGEOplutophase2"}, {"widgets": "SURFACEGEOplutocenterresolution,SURFACEGEOplutophase,surfacegeometrytargetname"}, {"order": "time1,opusid"}, {"view": "search"}, {"browse": "gallery"}, {"cart_browse": "gallery"}, {"startobs": 1}, {"cart_startobs": 1}, {"detail": ""}], "msg": None}
+        url = '/opus/__normalizeurl.json?SURFACEGEOpluto_centerresolution2=5&SURFACEGEOpluto_phase1=160&surfacegeometrytargetname=Pluto&qtype-SURFACEGEOpluto_phase=any&cols=opusid,instrument,time1,SURFACEGEOpluto_planetographiclatitude1,SURFACEGEOpluto_planetographiclatitude2,SURFACEGEOpluto_IAUwestlongitude1,SURFACEGEOpluto_IAUwestlongitude2,SURFACEGEOpluto_centerdistance,SURFACEGEOpluto_centerresolution,SURFACEGEOpluto_phase1,SURFACEGEOpluto_phase2&widgets=SURFACEGEOpluto_centerresolution,SURFACEGEOpluto_phase,surfacegeometrytargetname&order=time1,opusid&view=search&browse=gallery&cart_browse=gallery&startobs=1&cart_startobs=1&detail='
```

Comment:
I think you want to leave the old SURFACEGEO formats here (without the "_") in the URL so that we test normalizeurl's ability to switch from the old slug format to the new one.

Reply:
This wasn't entirely fixed so I went ahead and removed the remaining two "_" that shouldn't be there.